docs/wallet.md

```diff
@@ -1,5 +1,31 @@
 ## What is Drive wallet?
 
+Drive Wallet is a Google-powered cryptocurrency wallet, automatically created during registration, abstracting out private key management to a third party system - Google Drive.
+It allows a user to create/store private keys, transfer cryptocurrencies - all without having to install any software, understanding private/public keys.
+
+The goal of the wallet is to improve the usability and pave the road for widespread cryptocurrency adoption.
+
+The majority of the crypto community may not mind the complexities of entering in 42-character addresses or securing private keys, but the wider public does.
+Drive wallet automates accounts creations and private key management and makes the process of using cryptocurrencies smoother.
+
+The private keys are securely stored in user's Goolge Drive, **encrypted with a password provided by the user** . Specifically, they are store in the following locations:
```

Comment:
The private keys are securely stored in user's Goolge Drive and  **encrypted with a password provided by the user** . <del>Specifically, </del> They are stored in the following locations: